pages/braille.js

```diff
@@ -44,23 +44,21 @@ function BraillePoint({ allSelected, setSelected, number }) {
 export default function BraillePage() {
   const brailleNumbers = [1, 4, 2, 5, 3, 6]
   const [selected, setSelected] = useState(new Set())
-  const [solutionText, setSolutionText] = useState(new String(''))
-  const [solutionBraille, setSolutionBraille] = useState(new String(''))
-  const [solutionByRowsText, setSolutionByRowsText] = useState(new String(''))
+  const [entryPoints, setEntryPoints] = useState([])
+  const solutionText = entryPoints.map((entry) => decode(entry), '')
+  const solutionBraille = entryPoints.map((entry) => toUTF(entry), '')
+  const solutionByRowsText = entryPoints.map(
+    (entry) => decode(columns_to_rows(entry)),
+    ''
+  )
   const current = decode(selected)
 
   const handleSendButtonClick = () => {
-    const currentBraille = toUTF(selected)
-    const currentByRows = decode(columns_to_rows(selected))
-    setSolutionText(new String(solutionText + current))
-    setSolutionBraille(new String(solutionBraille + currentBraille))
-    setSolutionByRowsText(new String(solutionByRowsText + currentByRows))
+    setEntryPoints(entryPoints.concat([selected]))
   }
 
   const handleBackspaceButtonClick = () => {
-    setSolutionText(solutionText.slice(0, solutionText.length - 1))
-    setSolutionBraille(solutionBraille.slice(0, solutionBraille.length - 1))
-    setSolutionByRowsText(solutionByRowsText.slice(0, solutionByRowsText.length - 1))
+    entryPoints.pop()
```

Comment:
There should be setEntryPoints and I'm not sure if sending the same object causes re-render of the component